modular-docs-manual/files/sample-assemblies/modules/subassembly-1.adoc

```diff
@@ -0,0 +1,24 @@
+
+
+
+
+
+[id="subassembly-1"]
```

Comment:
**Andrew, Ben, or Dawn**, did we ever get an official ruling about designating subassemblies and housing them in the `modules` folder? We've had a need for this on our BA team and I think the idea was not well received but I don't recall a suggested alternative. Emily explains the scenario in her example, but essentially, a subassembly isn't necessarily just an assembly that can stand alone or be included within another assembly, in which case it would just be classified as an assembly. Instead, the need we've faced is to have subassemblies that never really stand alone but are a repeatable series of modules, more like a chapter (which Emily points out). So it's a handy way to have a reusable chapter-like set of modules, without having to repeat prerequisite and intro content the precedes the modules, and hence kept in the modules folder because it's treated more like a mega module than an assembly.

So, from a tooling side, any issues with this? I believe there was some rule that anything within `modules` folder must be a module, and by definition a module can't contain includes (which raises the question of included fragments/snippets, a separate question). So chapter-like subassemblies in this fashion, which we've found highly practical and useful, would be breaking that rule.

Would love your thoughts.

Reply:
Excellent point.